qiskit_ibm_runtime/estimator.py

```diff
@@ -229,13 +252,17 @@ class EstimatorV1(BasePrimitiveV1, Estimator, BaseEstimator):
 
     def __init__(
         self,
-        backend: Optional[Union[str, IBMBackend]] = None,
+        mode: Optional[Union[IBMBackend, Session, Batch]] = None,
+        backend: Optional[IBMBackend] = None,
```

Comment:
We don't actually need to change V1 primitive since we are deprecating it already